src/main/asciidoc/_chapters/hbase-default.adoc

```diff
@@ -2246,6 +2246,23 @@ The percent of region server RPC threads failed to abort RS.
 `false`
 
 
+[[hbase.regionserver.slowlog.systable.enabled]]
+*`hbase.regionserver.slowlog.systable.enabled`*::
++
+.Description
+
+      Should be enabled only if hbase.regionserver.slowlog.buffer.enabled is enabled.
+      If enabled (true), all slow/large RPC logs would be persisted to system table
+      hbase:slowlog (in addition to in-memory ring buffer at each RegionServer).
+      The records are stored in increasing order of time.
+      Operators can scan the table with various combination of ColumnValueFilter.
```

Comment:
Generally i feel that the querying will be like for given time range give me all the slow logs? Then they can drill down. So if they want to query with TS they can specify the Time range correct? Probably you can provide a sample input and output for that too.